packages/p2p/src/components/order-details/__test__/order-details.spec.js

```diff
@@ -0,0 +1,184 @@
+    order_information: { ...mock_order_info },
+    order_id: '123',
+    has_order_payment_method_details: false,
+    order_payment_method_details: [],
+    setOrderPaymentMethodDetails: jest.fn(),
+};
+const mock_sendbird_store = {
+    should_show_chat_on_orders: false,
+    setChatChannelUrl: jest.fn(),
+    createChatForNewOrder: jest.fn(),
+    registerEventListeners: jest.fn().mockReturnValue(jest.fn()),
+    registerMobXReactions: jest.fn().mockReturnValue(jest.fn()),
+};
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(() => ({
+        order_store: { ...mock_order_store },
+        sendbird_store: { ...mock_sendbird_store },
+    })),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isDesktop: jest.fn(() => true),
+}));
+
+jest.mock('Components/order-details/order-details-footer.jsx', () => jest.fn(() => <div>Order details footer</div>));
+
+jest.mock('Components/order-details/order-info-block.jsx', () => jest.fn(() => <div>Order Info Block</div>));
+
+jest.mock('Components/orders/chat/chat.jsx', () => jest.fn(() => <div>Chat section</div>));
+
+describe('<OrderDetails/>', () => {
+    it('should render component with loss of funds warning banner', () => {
+        useStores.mockReturnValue({
+            order_store: {
+                ...mock_order_store,
+                order_information: { ...mock_order_info, should_show_lost_funds_banner: true, chat_channel_url: null },
+            },
+            sendbird_store: { ...mock_sendbird_store },
+        });
+        render(<OrderDetails onPageReturn={jest.fn()} />);
+
+        expect(
+            screen.getByText(
+                'To avoid loss of funds, please do not use cash transactions. We recommend using e-wallets or bank transfers.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should render success message when highlight success is true', () => {
+        useStores.mockReturnValue({
+            order_store: {
+                ...mock_order_store,
+                order_information: { ...mock_order_info, should_highlight_success: true },
+            },
+            sendbird_store: { ...mock_sendbird_store },
+        });
+        render(<OrderDetails onPageReturn={jest.fn()} />);
+
+        expect(screen.getByText('Result str')).toBeInTheDocument();
+    });
+
+    it('should display footer info when show_order_footer is set', () => {
+        useStores.mockReturnValue({
+            order_store: {
+                ...mock_order_store,
+                order_information: { ...mock_order_info, should_show_order_footer: true },
+            },
+            sendbird_store: { ...mock_sendbird_store },
+        });
+        render(<OrderDetails onPageReturn={jest.fn()} />);
+
+        expect(screen.getByText('Order details footer')).toBeInTheDocument();
+    });
+
+    it('should display formatted currency when the torder is pending', () => {
```

Comment:
should display formatted currency when the **_torder_** is pending, maybe you want to write "order"?

Reply:
I'm so sorry for these many typos